emily/handler/src/api/models/common/mod.rs

```diff
@@ -50,6 +50,18 @@ pub enum Status {
     Failed,
 }
 
+impl std::fmt::Display for Status {
```

Comment:
In the signer crate we sometimes use `strum` to derive things like this for enums. Manually implementing this is fine, but I figured that it's good to know if we want to do things like this more often.

Reply:
I think [c47b9f2](https://github.com/stacks-sbtc/sbtc/pull/1625/commits/c47b9f2b277d1cf3c26828c97d27f5a6cdf9359d) resolves both your notes.